how-to-docs/define-rbac-policy.md

```diff
@@ -0,0 +1,37 @@
+# How to define a new RBAC policy
+
+Visit and clone the [acp-kube-rbac][acp kube rbac repo link] repo on GitLab.
+
+Create a new branch for your new policy and follow the instructions on the README in the repo.
+
+Documentation that can help write an RBAC policy can be found here: [RBAC Authorization][rbac docs].
+
+ * Please note that the docs linked above are only for reference.
+
+Also note the `rules` section of the policies in the GitLab repo are different to the ones that are in the linked docs. Each `apiGroup`/`resource`/`verb` is on its own line with a hyphen:
+
+[**policies/common/cluster-read-all.yaml**][cluster-read-all link]:
+```yaml
+...
+apiGroups:
+ - ""
+ - apps
+ - autoscaling
+ - batch
+ - extensions
+ - policy
+ - rbac.authorization.k8s.io
+resources:
+ - componentstatuses
+ - configmaps
+ - daemonsets
+ - deployments
+ - events
+...
+```
+
+Once you have created your new policy, push the new branch and create a pull request for it.
```

Comment:
Might also be best to say where you add your pr to i.e. slack

Reply:
Which slack channel do you think would be best for it? devops?